### PR TITLE
[DEVELOPER-4950] Make all topics link go to the stage blog categories page

### PR DESCRIPTION
**Pull request by kerrivader**

### CODE REVIEW

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/menu/menu--topics-menu.html.twig [hunk 1]
```diff
@@ -50,10 +50,10 @@
         {% endif %}
       </li>
     {% endfor %}
-      <li class="rhd-nav-all-topics"><a href="/blog/categories">All Topics<span class="icon"><i class="fas fa-caret-right"></i></span></a></li>
+      <li class="rhd-nav-all-topics"><a href="https://developers.stage.redhat.com/blog/categories/">All Topics<span class="icon"><i class="fas fa-caret-right"></i></span></a></li>
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/menu/menu--topics-menu.html.twig]: Why are we changing this to staging, just to change it back to prod? Could we change this up to go to the drupal url and let the export fix the rest up?

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/menu/menu--topics-menu.html.twig [hunk 2]
```diff
@@ -50,10 +50,10 @@
...
     </ul>
     <button class="previous-arrow slider-arrow"><i class="far fa-chevron-left"></i></button>
     <button class="next-arrow slider-arrow"><i class="far fa-chevron-right"></i></button>
-    <div class="rhd-nav-all-topics"><a href="/blog/categories">All Topics<span class="icon"><i class="fas fa-caret-right"></i></span></a></div>
+    <div class="rhd-nav-all-topics"><a href="https://developers.stage.redhat.com/blog/categories/">All Topics<span class="icon"><i class="fas fa-caret-right"></i></span></a></div>
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/menu/menu--topics-menu.html.twig]: Same here

**Comment by LightGuard** [comment]
@kerrivader any thoughts here?

**Comment by kerrivader** [comment]
@LightGuard I agree with you, but see this thread: https://issues.jboss.org/browse/DEVELOPER-4950
It doesn't go to a drupal url. It goes to a wordpress url, per my understanding.

**Comment by LightGuard** [comment]
@kerrivader Ugh. I hate hardcoding it but if we need it for the demo, fine.

**Pull request merged**